lua/plugins/init.lua

```diff
@@ -12,12 +15,12 @@ return {
   },
   { "gpanders/editorconfig.nvim", lazy = false },
```

Comment:
Are you good with me removing this? editorconfig support is built in in Neovim v0.9+ which is a requirement for AstroNvim v4 anyway.

Reply:
Oh, absolutely. Great to see it is built in now

Reply:
Ok, removed!

Reply:
@mehalter do I need to do something to enable editorconfig support? I'm having my files getting re-formated

Reply:
It seems that `indent_style = tab` is ignored

https://asciinema.org/a/4if3SHfE35lzIjkiR7NLcfRuB 

This seems like a neovim  issue and not a astronvim issue in any case

Reply:
This actually looks to be a language server issue. If you just use the tab key the indentation settings are set appropriately. Even with the editorconfig plugin you had before it behaves the same.

Reply:
what do you mean? I do use the tab key

Reply:
here @pipex https://asciinema.org/a/4IRuyK9QLjfYlSvyJrOMgBWpN

Reply:
why I am not suprised this is a TSServer issue? You are right that adding .prettierrc works :), thanks man